docs/tutorial/timeseries.rst

```diff
@@ -4,74 +4,46 @@
 
 .. _creating-timeseries:
 
 Creating a TimeSeries
 =====================
 
-A TimeSeries object can be created from local files.
-For convenience, **sunpy** can download several example time series of observational data.
-These files have names like ``sunpy.data.sample.EVE_TIMESERIES`` and ``sunpy.data.sample.GOES_XRS_TIMESERIES``.
-To create the sample `sunpy.timeseries.sources.goes.XRSTimeSeries`, type the following into your interactive Python shell:
+To create a `sunpy.timeseries.TimeSeries`,:
 
 .. code-block:: python
 
-    >>> import sunpy.timeseries as ts
+    >>> import sunpy.timeseries
     >>> import sunpy.data.sample  # doctest: +REMOTE_DATA
-    >>> my_timeseries = ts.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES)  # doctest: +REMOTE_DATA
-
-This is calling the `~sunpy.timeseries.TimeSeries` factory to create a time series from a GOES XRS FITS file.
-
-The variable ``my_timeseries`` is a `~sunpy.timeseries.GenericTimeSeries` object.
-To create one from a local GOES/XRS FITS file try the following:
-
-.. code-block:: python
-
-    >>> my_timeseries = ts.TimeSeries('/mydirectory/myts.fits', source='XRS')   # doctest: +SKIP
-
-**sunpy** will attempt to detect automatically the instrument source for most FITS files.
-However time series data are stored in a variety of file types (FITS, txt, csv, CDF), and so it is not always possible to detect the source.
-**sunpy** ships with a number of known instrumental sources, and can also load CDF files that conform to the `Space Physics Guidelines for CDF <https://spdf.gsfc.nasa.gov/sp_use_of_cdf.html>`__.
-If you would like **sunpy** to include another instrumental source see the `Newcomers' Guide <https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html>`__.
-
-The `~sunpy.timeseries.TimeSeries` factory has the ability to create a list of TimeSeries objects using a list of filepaths, a folder or a glob, for example:
-
-.. code-block:: python
-
-    >>> my_ts_list = ts.TimeSeries('filepath1', 'filepath2', source='XRS')   # doctest: +SKIP
-    >>> my_ts_list = ts.TimeSeries('/goesdirectory/', source='XRS')   # doctest: +SKIP
-    >>> my_ts_list = ts.TimeSeries(glob, source='XRS')   # doctest: +SKIP
-
-When manually specifying the source this functionality will only work with files from the same single source, generating a source specific child of the `~sunpy.timeseries.GenericTimeSeries` class such as the `~sunpy.timeseries.sources.goes.XRSTimeSeries` above.
+    >>> sunpy.data.sample.GOES_XRS_TIMESERIES  # doctest: +REMOTE_DATA
+    PosixPath('.../go1520110607.fits')
+    >>> my_timeseries = sunpy.timeseries.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES)  # doctest: +REMOTE_DATA
 
-Instead of creating a list of one TimeSeries object per file, you can create a single time series from multiple files using the keyword argument ``concatenate=True``:
+In many cases, sunpy will automatically detect the type of the file as well as the instrument associated with it.
```

Comment:
```suggestion
In many cases for which their is a specific client, sunpy will automatically detect the type of the file as well as the instrument associated with it.
```

Reply:
maybe we could also link to a list of the data sources that sunpy can actually parse?

Reply:
I think that is not within scope of this tutorial. I also would like to avoid using the words "client" or "factory". That language only makes sense in the context of our implementation of how we detect specific sources which the user, especially at this level, does not need to know about.